views/alumini.ejs

```diff
@@ -107,9 +107,16 @@
                   <input type="text" class="form-control" id="minor" placeholder="Minor" required>
                 </div>
               </div>
+              <div class="form-group">
+                <label for="bType">Beacon Type</label>
+                <select class="form-control" id="bType">
+                  <option>RUUVI</option>
+                  <option>AXA</option>
+                </select>
+              </div>
               <div class="form-group">
                 <label for="macAddress">MAC Address</label>
-                <input type="text" class="form-control" id="macAddress" placeholder="MAC Address" required>
+                <input type="text" class="form-control" id="macAddress" placeholder="MAC Address" pattern="^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})$" title="XX:XX:XX:XX:XX" required>
```

Comment:
Assign every input a `name` attribute. For example this would be called `beacon_id`. This should match the key in the API you want to send. See my above comment on the format of the API body since it will be used to generate the JSON to send to the api.